images/automation-broker-apb.yml

```diff
@@ -16,10 +16,3 @@ owners:
 - jesusr@redhat.com
 - jlanford@redhat.com
 - fabian@redhat.com
-content:
-  source:
```

Comment:
note that where there is no upstream source, ART will need to remove the `gating.yaml` manually from dist-git or it will stay.